graphql_api/tests/test_repository.py

```diff
@@ -424,3 +426,16 @@ def test_repository_not_found(self):
             "__typename": "NotFoundError",
             "message": "Not found",
         }
+
+    def test_repository_has_ats_configured(self):
+        repo = RepositoryFactory(
+            author=self.owner,
+            active=True,
+            private=True,
+            yaml={
+                "flag_management": {"individual_flags": {"carryforward_mode": "labels"}}
```

Comment:
I'd suggest adding tests for an empty yaml or when flag_management isn't in the yaml

Reply:
The other tests check for this situation and I have updated those to reflect that. See lines 126, 177.